recipes/partd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.3.4" %}
+
+package:
+  name: partd
+  version: {{ version }}
+
+source:
+  fn: partd-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/partd/partd-{{ version }}.tar.gz
+  md5: 6581f265299ceaf1df86422fc8c0a754
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - locket
```

Comment:
Because this depends on locket you will need to re-start the CIs in the feedstock once we merge this because `locket` won't be in the channel before the first build of `partd`.